homeassistant/components/google_air_quality/sensor.py

```diff
@@ -0,0 +1,186 @@
+        value_fn=lambda x: x.pollutants.no2.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="o3",
+        translation_key="ozone",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.o3.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="pm10",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PM10,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        value_fn=lambda x: x.pollutants.pm10.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="pm25",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PM25,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        value_fn=lambda x: x.pollutants.pm25.concentration.value,
+    ),
+    AirQualitySensorEntityDescription(
+        key="so2",
+        translation_key="sulphur_dioxide",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_BILLION,
+        value_fn=lambda x: x.pollutants.so2.concentration.value,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: GoogleAirQualityConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up sensor platform."""
+    coordinator = entry.runtime_data
+    entities: list[SensorEntity] = []
+
+    entities.extend(
+        AirQualitySensorEntity(coordinator, description)
+        for description in AIR_QUALITY_SENSOR_TYPES
+        if description.exists_fn(coordinator.data)
+    )
+    async_add_entities(entities)
+
+
+class AirQualitySensorEntity(
+    CoordinatorEntity[GoogleAirQualityUpdateCoordinator], SensorEntity
+):
+    """Defining the Air Quality Sensors with AirQualitySensorEntityDescription."""
+
+    entity_description: AirQualitySensorEntityDescription
+    config_entry: GoogleAirQualityConfigEntry
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: GoogleAirQualityUpdateCoordinator,
+        description: AirQualitySensorEntityDescription,
+    ) -> None:
+        """Set up Air Quality Sensors."""
+        super().__init__(coordinator)
+        self.entity_description = description
+        name = f"{self.coordinator.config_entry.data[CONF_LATITUDE]}_{self.coordinator.config_entry.data[CONF_LONGITUDE]}"
+        self._attr_unique_id = f"{description.key}_{name}"
+        self.coordinator = coordinator
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, name)},
+            name=name,
+            entry_type=DeviceEntryType.SERVICE,
+        )
+        self._attr_translation_placeholders = {
+            "local_aqi": coordinator.data.indexes[1].display_name
+        }
```

Comment:
What's this?

Reply:
This is the local AQI for the country, in which the selected coordinates are. 
For Dutch it would for example be: LKI (NL)
https://developers.google.com/maps/documentation/air-quality/laqis?hl=de